server/index.js

```diff
@@ -1,21 +1,26 @@
+import dotenv from "dotenv";
 import express from "express";
 import bodyParser from "body-parser";
 import mongoose from "mongoose";
 import cors from "cors";
 
-import postRoutes from "./routes/posts.js";
+import userRoutes from "./routes/users.js";
 
 const app = express();
+const config = dotenv.config()
+if (config.error){
+  throw config.error;
+}
+const env = config.parsed;
 
 app.use(bodyParser.json({ limit: "30mb", extended: true }));
 app.use(bodyParser.urlencoded({ limit: "30mb", extended: true }));
 app.use(cors());
 
-app.use("/posts", postRoutes); //every route inside of post routes will start with post
+app.use("/users", userRoutes); //every route inside of post routes will start with post
 
-const CONNECTION_URL =
-  "mongodb+srv://Jonathan:Jonathan123@cluster0.2rpdj.mongodb.net/<dbname>?retryWrites=true&w=majority";
-const PORT = process.env.PORT || 5000;
```

Comment:
does this effect everyone connecting to the same db?

Reply:
We could use an account for "prod" so everyone can test ans use the same database. Using env will help if any of us want to use another key as playground